proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -221,20 +213,3 @@ message UpdateUserSettingsRequest {
 }
 
 message UpdateUserSettingsResponse {}
-
-message GetUserPluginPreferencesRequest {
-  reserved 1;
-  reserved "user_id";
-}
-
-message GetUserPluginPreferencesResponse {
-  UserPluginPreference preference = 1;
```

Comment:
Did we remove this as well, the `UserPluginPreference` or is that used?

Reply:
It's not used actually. It can be removed also: https://github.com/bufbuild/buf/pull/3674